pages/searching/time_frame_selector.rst

```diff
@@ -1,38 +1,96 @@
-Time frame selector
+Time Frame Selector
 -------------------
 
-The time frame selector defines in what time range to search in. It offers three different ways of selecting a time range and
-is vital for search speed: If you know you are only interested in messages of the last hour, only search in that time frame.
-This will make Graylog search in :doc:`relevant indices </pages/configuration/index_model>` only and greatly reduce system load and required resources.
+The Time frame selector allows you to pull specific time ranges from your Graylog data. 
+This feature will help you and other Graylog users analyze issues that affect multiple aspects 
+of your environment. Most importantly, the tool offer multiple ways to filter time ranges 
+you need for success.
 
-.. image:: /images/searching/queries_time_range_selector.png
-   :align: center
+.. image:: /images/searching/time_frame_selector_wide.png
+
+
+To that end, the tool's ability to help you queries can help you perform such actions as:
```

Comment:
```suggestion
To that end, the tool helps you to build queries can help you perform such actions as:
```
This part was a little unclear to me, so I am wondering if this wording helps.

Reply:
Yeah, the possessive is weird since a "tool" is not a person. Your suggestion works.